database/scripts/20220718_210029_fecha_estimada_ofertas_up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE plan_adquisiciones."Registro_plan_adquisiciones"
+ADD COLUMN IF NOT EXISTS fecha_estimada_ofertas_inicio TIMESTAMP WITH TIME ZONE;
+
+ALTER TABLE plan_adquisiciones."Registro_plan_adquisiciones"
+ADD COLUMN IF NOT EXISTS fecha_estimada_ofertas_fin TIMESTAMP WITH TIME ZONE;
```

Comment:
```suggestion
ADD COLUMN IF NOT EXISTS fecha_estimada_ofertas_fin TIMESTAMP WITH TIME ZONE;

```

Adicionalmente, falta agregar y/o actualizar comentarios a las columnas ([ejemplo](https://stackoverflow.com/a/32071293))
Las 2 fechas existentes tienen los siguientes comentarios, que serían ambigüos una vez se creen las dos columnas:
![image](https://user-images.githubusercontent.com/6588872/179897458-29c88f91-0b24-4745-bbd7-a96e0eb2bbbf.png)

Se podría tomar los que están... :
- Fecha estimada de inicio del proceso
- Fecha estimada de finalizacion del proceso

... pero según el rango de fechas del que se trate, complementar "del-proceso-de-selección/de-presentacion-de-ofertas" combinar según sea el caso

Por ejemplo, como en la primer migración, después de haber creado una tabla:
https://github.com/udistrital/plan_adquisiciones_crud/blob/8bce27e2a2cdd6b1d402fa42811e8b45d1488455/database/scripts/tablas_plan_adquisiciones.up.sql#L40-L55

(Sobra decir que no es necesario reversar los comentarios en el down.sql para las columnas creadas en el up.sql)